src/Fantomas.Tests/FunctionDefinitionTests.fs

```diff
@@ -526,14 +518,12 @@ let private addTaskToScheduler (scheduler : IScheduler) taskName taskCron prio (
 """  ({ config with PageWidth = 100 })
     |> prepend newline
     |> should equal """
-let private addTaskToScheduler
-    (scheduler: IScheduler)
-    taskName
-    taskCron
-    prio
-    (task: unit -> unit)
-    groupName
-    =
+let private addTaskToScheduler (scheduler: IScheduler)
+                               taskName
+                               taskCron
+                               prio
+                               (task: unit -> unit)
+                               groupName =
```

Comment:
@nojaf shouldn't the equals character here be in the next line? this way if someone adds a new parameter, only 1 line needs to change (instead of 2)

Reply:
Yeah I thought about it, but that would be inconsistent with how Microsoft formats member definitions. There the equals sign is not on a new line.

Reply:
I don't think you're interpreting correctly (from the snippet I posted). The rule could be that the equals line is in the same line where the return type is (if there's no return type, then the equals is left alone).

Reply:
and I just realised that the snippet I was referring to (https://github.com/dotnet/docs/issues/18806#issuecomment-651890110) actually has the equal char in its own line, even when there is a return type, but I didn't mean to do that; I think the equals char can stay in the same line as the return type in that case

Reply:
Well, I understood it as: if there is a return type, it and the equal sign can go to the next line.
Following the same logic as what was written [here](https://github.com/dotnet/docs/pull/18793/files#diff-eed63f6624dbb527e2e9533697f1ff6bR112).

Reply:
Updated according to the conversation in https://github.com/dotnet/docs/issues/18806